### ISSUE TITLE
MariaDB Support?

### ISSUE THREAD

**Issue by robertsilen** [collectd/collectd#4323]
Hello, I noticed that the [documentation](https://www.collectd.org/documentation/) and the github [README](https://github.com/collectd/collectd/blob/main/README.md) only mention MySQL. Considering most distros run MariaDB by default, is there any particular reason MariaDB couldn't be mentioned? Only seems fair, and would remove any compatibility doubts. I could make a PR suggestion with some changes to include MariaDB?

**Comment by eero-t** [comment]
Yes, PR would be appreciated!

(Along with checking that `mysql` plugin works with MariaDB as expected, if you are not already doing that.)

**Comment by robertsilen** [comment]
Hello, MariaDB has been default in distros, e.g. Debian since 2017 and Fedora since 2015, that effectively any compatibility issues with collectd would already have turned up already (and been fixed). I suggested a PR https://github.com/collectd/collectd/pull/4329 to add MariaDB to the README, and I added a mention of MariaDB to the Wiki about [Plugin MySQL](https://github.com/collectd/collectd/wiki/Plugin-MySQL). Fairly simple additions, it could be explained more, but I think this already works so people understand the compatibility. I did no find the location of the manpages in the source, where is that?
